android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/responses/GetVehicleDataResponseTests.java

```diff
@@ -139,7 +139,7 @@ public void testJson() {
         try {
             //set up the JSONObject to represent GetVehicleDataResponse
             //TIRE_PRESSURE
-            tireStatusObj.put(TireStatus.KEY_PRESSURE_TELL_TALE, VehicleDataHelper.TIRE_PRESSURE_TELL_TALE);
```

Comment:
@kostyaBoss I would suggest reverting these changes in `GetVehicleDataResponseTests.java`. These are valid changes, since the `pressureTellTale` functions are being renamed per the parameter, but is out of scope of the PR.